bciers/apps/administration/tests/components/facilities/FacilityPageBanner.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { render, screen } from "@testing-library/react";
+import * as nextNavigation from "next/navigation";
+import FacilityPageBanner from "apps/administration/app/components/facilities/FacilityPageBanner";
+
+vi.mock("next/navigation", () => {
+  const actual = vi.importActual<typeof nextNavigation>("next/navigation");
+  return {
+    ...actual,
+    useSearchParams: vi.fn(),
+  };
+});
```

Comment:
nit: You could also use an existing mock for consistency `import { useSearchParams } from "@bciers/testConfig/mocks";`